### ISSUE TITLE
Documentation missing Wrappers

### ISSUE THREAD

**Issue by JoyMonteiro** [mcgibbon/sympl#44]
Currently, the documentation do not mention Wrappers or how to use them. This must be fixed.

**Comment by mcgibbon** [comment]
I agree. This is briefly mentioned in the quickstart, but it should have its own section.
